dc-cudami-server/dc-cudami-server-backend-lobid/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>de.digitalcollections.cudami</groupId>
+    <artifactId>dc-cudami-server</artifactId>
+    <version>3.7.0-SNAPSHOT</version>
+  </parent>
+
+  <name>DigitalCollections: cudami Repository Server (Backend IMPL External System lobid.org)</name>
+  <artifactId>dc-cudami-server-backend-lobid</artifactId>
+  <packaging>jar</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>de.digitalcollections.cudami</groupId>
+      <artifactId>dc-cudami-server-backend-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>de.digitalcollections.cudami</groupId>
+      <artifactId>dc-cudami-lobid-client</artifactId>
+      <version>3.7.0-SNAPSHOT</version>
```

Comment:
läßt sich das noch in properties auslagern? Oder kann man gleich die project.version verwenden?